frontend/src/pages/OrganizationMembersPage.tsx

```diff
@@ -25,7 +25,7 @@ export const OrganizationMembersPage: React.FC = () => {
       owner: selectOwner(state),
     }
   })
-  const enterprise = !license?.plan?.billing
+  const enterprise = !!license && !license.plan.billing
```

Comment:
Yes this is correct. It changed it from just not having a billing plan to having a plan and not having a billing plan. That way if you just don't have a plan you aren't an enterprise user. All the ! is so it alway returns a boolean instead of undefined.